### PR TITLE
Replace simple_captcha with invisible_captcha

### PR DESCRIPTION
**Pull request by kikito**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Frontend tools used include [SCSS](http://sass-lang.com/) over [Foundation](http
 
 ## Configuration for development and test environments
 
-Prerequisites: install git, ImageMagick, Ruby 2.2.3, bundler gem, ghostscript and PostgreSQL (>=9.4).
+Prerequisites: install git, Ruby 2.2.3, bundler gem, ghostscript and PostgreSQL (>=9.4).
```
Comment (xuanxu) [inline README.md]: :tada:

**Comment by voodoorai2000** [comment]
Mr. 🤖

**Comment by xuanxu** [comment]
:tada: :clap: :clap:

**Comment by markets** [comment]
🔝  👏 

Don't hesitate to contact me to discuss about: updates, changes, fixes, feedback, new features...

Thanks @kikito for all your help with latest release (and the incoming one)!

**Pull request merged**